src/components/welcome-landing/welcome-landing.tsx

```diff
@@ -1,61 +1,36 @@
 'use client'
 
-import {getSteamConfig} from "@/services/getSteamConfig";
-import {Alert, Button, Link as MatLink, List, ListItem, ListItemAvatar, ListItemText} from "@mui/material";
+import {Button,} from "@mui/material";
 import Link from "next/link";
-import AccountCircleIcon from '@mui/icons-material/AccountCircle';
-import VpnKeyIcon from '@mui/icons-material/VpnKey';
-import {useRouter} from "next/navigation";
-import React from "react";
+import {useSession} from "next-auth/react";
 
-export default function WelcomeLanding(): React.ReactElement {
-    const config = getSteamConfig();
-    const appConfigured = config.apiKey && config.userId;
+export default function Home() {
+    const session = useSession()
 
-    const router = useRouter();
+    const sessionLayout = <div>
+        <ul>
+            <li>Session status: {session.status}</li>
+            <li>
+                Session data:
+                <pre>{JSON.stringify(session.data, null, 2)}</pre>
```

Comment:
Should we remove this debug info before we merge?

Reply:
leaving as is for now. next pr will remove it with a proper log in button and show some sort of info when logged in.